requirements.txt

```diff
@@ -1,4 +1,7 @@
+chainmap
```

Comment:
Looks like it might be necessary and seems to work for both 2 and 3. The original way didn't't work for python 2. More info:
[chainmap](https://pypi.python.org/pypi/chainmap)
[chainMapImportError](https://www.reddit.com/r/learnpython/comments/5c2i8s/importerror_cannot_import_name_chainmap/)